yorkie/src/main/kotlin/dev/yorkie/document/json/JsonTree.kt

```diff
@@ -170,9 +170,14 @@ public class JsonTree internal constructor(
 
         val ticket = context.lastTimeTicket
         val crdtNodes = if (contents.firstOrNull()?.type == DEFAULT_TEXT_TYPE) {
-            val compVal = contents
-                .filterIsInstance<TextNode>()
-                .joinToString("") { it.value }
+            val textNodes = contents.filterIsInstance<TextNode>()
+            val compVal = if (textNodes.size == 1) {
+                textNodes.single().value
+            } else {
+                textNodes.joinTo(StringBuilder(textNodes.sumOf { it.value.length }), "") {
+                    it.value
+                }.toString()
+            }
```

Comment:
Default `joinToString` for large text can result in multiple resizings of the list,
so I changed  to `joinTo` with initial size that matched the text length.